src/components/Button.jsx

```diff
@@ -1,9 +1,17 @@
-import styles from "../partials/_button.module.scss";
+import styles from '@/partials/_button.module.scss';
 
-export default function Button({ content, variant, fun }) {
+export default function Button({
+  variant = 'default',
+  size = 'sm',
```

Comment:
Set default values for the props so you no longer need to explicitly include these props when using the component elsewhere